### ISSUE TITLE
API proposal: currentRefinedValues widget

### ISSUE THREAD

**Issue by Jerskouille** [algolia/instantsearch.js#404]
Follow up of #112 and #378:

```js
search.addWidget(
  instantsearch.widgets.currentRefinedValues({
    container: '#current-refinements-list',
    cssClasses: {root, header, body, clearAll, list, item, link, count, footer},
    attributes: [ // Optional, if not specified, takes all of the facets
      { name: 'category'[, transformData: function (data) { /* ... */ return data; }]},
      { name: 'brand'[, transformData: function (data) { /* ... */ return data; }]}
    ],
    templates: {
      header: '',
      item: '' + // Called with this.facets[current_facet.name].transformData({cssClasses, facetName, name, count})
        '{{ name }}' +
        '<span class="{{ cssClasses.count }}">' +
        '  {{ count }}' +
        '</span>',
      clearAll: 'Clear all'
      footer: ''
    },
    clearAll: 'after', // [true, false, 'before', 'after'] (true == 'before')
    hideContainerWhenNoRefinements: true
  })
);
```

**Comment by Jerskouille** [comment]
I've changed `facets` with `attributes` since it should also handle `_tags`.

**Comment by vvo** [comment]
Moving to "next" (1.1.0)

**Comment by redox** [comment]
@Jerskouille that's done, right?

**Comment by Jerskouille** [comment]
Oh yes, merged in #700 .
